invenio_communities/communities/services/config.py

```diff
@@ -72,6 +74,17 @@ class SearchOptions(SearchOptionsBase, SearchOptionsMixin):
     ]
 
 
+def children_allowed(record, _):
+    """Determine if children are allowed."""
+    if isinstance(record, dict):
+        return record.get("children", {}).get("allow", False)
+    elif isinstance(record, SimpleNamespace):
+        if hasattr(record, "children"):
+            return getattr(record.children, "allow", False)
+
+    return False
+
+
```

Comment:
I think we have to fix the root of the problem:

https://github.com/inveniosoftware/invenio-communities/blob/f56b388de22c7470dd741ca713b644f3cb137c03/invenio_communities/communities/entity_resolvers.py#L38-L41

```python
    fake_community_obj = SimpleNamespace(
        id=community_dict["id"],
        slug=community_dict["slug"],
       children=community_dict.get("children", {})
    )
```
(please confirm whether `community_dict` has `children` in it, it might be it has to be added somewhere)

...

and then safely get the `children.allow` property:

```python
return getattr(community.children, "allow", False)
```